src/nvim/option.c

```diff
@@ -4364,6 +4365,8 @@ static void check_redraw(uint32_t flags)
     changed_window_setting();
   if (flags & P_RBUF)
     redraw_curbuf_later(NOT_VALID);
+  if (flags & P_RWINONLY)
```

Comment:
`make clint` will complain about the missing braces. We _always_ uses braces. Now for the purpose of steadily changing the code to fit our coding conventions, `make clint` also looks at the surrounding lines. You probably will have to add braces for the other `if` as well.